test/recur_iterator_test.js

```diff
@@ -1261,6 +1261,15 @@ suite('recur_iterator', function() {
           '2026-03-14T01:00:00',
         ]
       });
+
+      testRRULE('FREQ=YEARLY;BYYEARDAY=-275,-276,-277,-278,-279,-280,-281;BYDAY=MO', {
```

Comment:
I think we also should test some positive `BYYEARDAY`s here, and some that cross February 29th, to be sure everything is correct in those cases.